networks/movement/movement-full-node/src/node/tasks/execute_settle.rs

```diff
@@ -110,9 +110,10 @@ where
 			}
 		};
 
+		let block_id_hex = hex::encode(&block_id);
 		info!(
-			block_id = %hex::encode(block_id.clone()),
-			da_height = da_height,
+			block_id = block_id_hex,
```

Comment:
`target: "movement_telemetry"` is missing here too. What's the logic for including/not including?